document-store/src/integrationTest/java/org/hypertrace/core/documentstore/DocStoreTest.java

```diff
@@ -981,6 +982,23 @@ public void testDelete(String dataStoreName) throws IOException {
     Assertions.assertEquals(collection.count(), 0);
   }
 
+  @ParameterizedTest
+  @MethodSource("databaseContextProvider")
+  public void testBulkDelete(String dataStoreName) throws IOException {
```

Comment:
Runs for both postgres and mongo, right?

Reply:
yes